docs/guides/postgres/remote-replica/monitoring.md

```
@@ -0,0 +1,176 @@
+title: Monitoring PostgreSQL Remote Replicas
+menu:
+  docs_{{ .version }}:
+    identifier: pg-remote-replica-monitoring
+    name: Monitoring
+    parent: pg-remote-replica
+    weight: 30
+menu_name: docs_{{ .version }}
+section_menu_id: guides
+---
+
+> New to KubeDB? Please start [here](/docs/README.md).
```

markdownlint-cli2 (0.23.2)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)